libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -416,7 +496,8 @@ class AC_AttitudeControl {
     virtual float get_roll_trim_cd() { return 0;}
 
     // passthrough_bf_roll_pitch_rate_yaw - roll and pitch are passed through directly, body-frame rate target for yaw
-    virtual void passthrough_bf_roll_pitch_rate_yaw(float roll_passthrough, float pitch_passthrough, float yaw_rate_bf_cds) {};
+    virtual void passthrough_bf_roll_pitch_rate_yaw_cd(float roll_passthrough_cd, float pitch_passthrough_cd, float yaw_rate_bf_cds) {};
```

Comment:
This one could be implemented as pass through like the others, and not virtual. It should save flash on copter, even tho its never used. The compiler has a hard time telling if it can remove virtual functions, so we might be paying for this anyway.